### ISSUE TITLE
Could you please give me an advice on exporting jp result as table?

### ISSUE THREAD

**Issue by cbalzani** [DanChaltiel/nih.joinpoint#7]
Hi, Dan!

It's me again. Thanks for the constant support you provide for the users of your great package.

This time, I am writing to ask your advice and to ask for more information about the info in name(jp).

Important info about my dataset and joinpoint settings:
- I am using years as the x variable, ranging from 2000 to 2020.
- I have 100 clusters which I am using as the "by" argument.

Therefore, as you may imagine, I cannot display all plots at once to assess the result. I want to export it as a table and then only plot significant clusters. Any suggestions? Anything is welcome :)

What I am thinking:

The most important information for me is the significance of the final selected model, the number and years of joinpoint and APC. A plus would be the significance of non-selected models, intercept value and MSE.

Therefore, my current idea is to create a sequence of unique values for the jp (not sure yet how to do this, tho). And then merge the jp$report and jp$apc based on the unique value and export the result as CSV. What would you think about that?

Bonus question!! 🎉

Could you please recommend materials to understand better the outcomes of jp? For example: in jp$report, what the df mean? or the auto_corr? and the difference between joinpoin_95_lcl and _ucl?

Well, that's all. Thank you very much!!

Best,
C.

**Comment by DanChaltiel** [comment]
Hi C.

As you can see in `help(jp_point)`, you can use `by_level` to filter the plots you want to keep.

For instance, you can write something like this:

```
run_opt = run_options(model="ln", max_joinpoints=3, n_cores=3)
jp = joinpoint(nih_sample_data, x=year, y=rate, by=sex, se=se,
               run_opts=run_opt)
MSE_LIMIT = 1.4 #you set your own condition
x = jp$report %>% 
  distinct(cluster=sex, mse) %>% 
  filter(mse>MSE_LIMIT)
jp_plot(jp, by_level=x$cluster)
```

However, trying out this code made me realize a tiny bug that prevented it to work. I fixed it in 246dc44 so you will have to update the package to use it.

For the bonus question, the outcome of `jp` is unfortunately not much detailed in the original NIH Joinpoint Software so I unfortunately won't be able to help much with that. Maybe you can get more info on their website. My guess is that `df` is "degree of freedom" as usual, and that `joinpoin_95_lcl` and `_ucl` are limits of some sort of confidence interval though (lower vs upper). No clue about `auto_corr`, maybe there is an autocorrelation term in the intern model, like in mixed models?

**Comment by DanChaltiel** [comment]
In the absence of response, I guess I can close this issue now.
